### PR TITLE
doc Replace graph stage and processing stage by operator in .md files for stream

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #24995
Only taking care of the phrases "graph stage" and "processing stage".

More granular words, "graph" and "stage" will be taken care in other PRs.

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/stream-customize.md [hunk 1]
```diff
@@ -181,9 +181,9 @@ and `abortReading()`
 
 An example of how this API simplifies a stage can be found below in the second version of the `Duplicator`.
 
-### Custom linear processing stages using GraphStage
+### Custom linear operators using GraphStage
 
-Graph stages allows for custom linear processing stages through letting them
+GraphStage allows for custom linear processing stages through letting them
 have one input and one output and using `FlowShape` as their shape.
```
Comment (richardimaoka) [inline akka-docs/src/main/paradox/stream/stream-customize.md]: Here I converted "Graph stages" to "GraphStage".  
Actually does the sentence look little weird? and should be re-written like "To define custom linear operators, you should extend GraphStage using `FlowShape` which has one input and one output." ?

**Review by ktoso — APPROVED** [review]
Excellent, thanks a ton!

File: akka-docs/src/main/paradox/stream/stream-customize.md (hunk 1 above)
Comment (ktoso) [reply akka-docs/src/main/paradox/stream/stream-customize.md]: yeah good suggestion, please reword and link to stream-customize from the GraphStage, let's  merge once that is done

**Review by chbatey — APPROVED** [review]
Couple of nits then LGTM

One thing i find weird is when reading is the use of operator when describing a running process but as long as it is consistent I think that is fine

File: akka-docs/src/main/paradox/stream/stream-customize.md (hunk 1 above)
Comment (richardimaoka) [reply akka-docs/src/main/paradox/stream/stream-customize.md]: Sure, updated the sentence. stream-customize.md is this file, so I'm not linking from `GraphStage` here.

**Comment by richardimaoka** [comment]
Reflected review comments.

**Review by raboof — APPROVED** [review]
Nice simplification, thanks!

**Pull request merged**